### PR TITLE
adds in default url options

### LINKED ISSUE #36: Error on User Create - Assuming Devise Issue
I'm getting an error when creating users that links back to not setting a default_host with devise. Anyone else seeing this?

```
Devise::Mailer#confirmation_instructions: processed outbound mail in 51.2ms
ActionView::Template::Error: Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true
```

In [devise setup](https://github.com/plataformatec/devise#getting-started), they recommend setting: 
`config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }`

in `config/environments/development.rb`

### PR DESCRIPTION
**Pull request by troystarwalt**
Resolves #36

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@
 
   config.action_mailer.perform_caching = false
 
+  # Setting URL Options.
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
```
Comment (pachonk) [inline config/environments/development.rb]: Can we remove 1 of these 2 lines please?

File: config/environments/development.rb [hunk 2]
```diff
@@ -31,6 +31,10 @@
 
   config.action_mailer.perform_caching = false
 
+  # Setting URL Options.
```
Comment (pachonk) [inline config/environments/development.rb]: Setting Mailer URL Options

Comment (pachonk) [inline config/environments/development.rb]: Also, no period (but that's a person pet peeve)

**Comment by pachonk** [comment]
@troystarwalt Your email in your git config is incorrect making it impossible to link these commits to your github account.
You might want to fix that so these commits ultimately get linked to your github account.
If you want the email privacy, github had a users.github.noreply style email you can use.

**Comment by troystarwalt** [comment]
@pachonk I'll take a look at my settings. Thank you!

**Pull request merged**